<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visi√≥n Artificial</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Capture</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="cornersFrame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="harrisF_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Max. difference&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="showFloatingRange_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Floating range</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="corners_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Bottom-up segmentation&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="resizeButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize Image</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showBottomUp_checkbox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom-up</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>captureButton</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>resizeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
